### PR TITLE
Readme release review

### PR DESCRIPTION
**Pull request by evanfloden**
First update of the README for release. 

We need to change the hierarchy of the Getting Started tutorial.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,163 +1,257 @@
...
-1. Download GraalVM (Java 11 version) from [this link](https://github.com/graalvm/graalvm-ce-builds/releases/tag/vm-20.2.0).
+The key features of the Tower CLI are:
 
-2. Install `native-image` tool:
+- **A Nextflow-like experience**: Tower CLI provides a developer friendly environment. Workflows can be launched similar to Nextflow but with the benefits of Tower such as monitoring, logging, resource provisioning, dataset management and collaborative sharing.
 
-    ```
-    gu install native-image
-    ``` 
+- **Infrastructure as Code**: All Tower resources including Workspaces and Compute Environments can be described in a declarative manner using YAML or JSON. This allows a complete definition of an analysis environment that can be versioned and treated as code. It greatly simplifies sharing, re-using and administration.
```
Comment (jordeu) [inline README.md:16]: Current version, the only resources that the CLI can import/export are launchpad pipelines and compute environments. And YAML is not supported, only JSON.

File: README.md [hunk 2]
```diff
@@ -1,163 +1,257 @@
...
 
 ```
-tw launch nextflow-io/hello
+$ tw launch nextflow-io/hello
 ```
 
-Update a launchpad pipeline
+### :gift: Bonus: Activate autocomplete 
+
+Install autocomplete in your current session, type `tw pi` then press tab twice.
```
Comment (jordeu) [inline README.md:171]: Here I understand that I've to:

1. type `tw pi`
2. press tab twice
3. Execute `source <(tw generate-completion)`

Maybe it's better to first show the command to run and then explain how to use autocomplete.

File: README.md [hunk 3]
```diff
@@ -1,163 +1,257 @@
...
-tw launch nf-core/sarek --compute-env "aws seqera" --profile test,docker
+$ tw launch nf-core/sarek --compute-env "aws-ce" --profile test,docker
 ```
 
-### Development
+
+Build binary development version
+--------------------------------
+
+The Tower CLI is based on a Java native compilation, Micronaut HTTP declarative client and Tower domain classes.
```
Comment (jordeu) [inline README.md:210]: We need to update this sentence to something like: "Tower CLI is based on a Java GraalVM native compilation to produce a platform binary executable."

File: README.md [hunk 4]
```diff
@@ -1,163 +1,257 @@
...
+    $ ./gradlew nativeImage
+    ```
+
+4. then run
+
+    ```
+    $ ./build/graal/tw
+    ```
+
+Development
```
Comment (jordeu) [inline README.md:232]: We can remove this section, we are no longer using SNAPSHOT dependencies.

File: README.md [hunk 5]
```diff
@@ -1,163 +1,257 @@
...
+-------------------------------------
+
+You can run a non-binary development version running the `./tw` script in the root of this repository.
+
+License
+-------
+
+[Mozilla Public License v2.0](https://github.com/seqeralabs/tower-cli/blob/master/LICENSE.txt)
+
 ### Credits & Links
```
Comment (jordeu) [inline README.md:251]: Not sure if we want to keep this links, I think I'd completely remove this section. We only used them for the first prototype. In fact some of them are a bit outdated and we are not using what they mention, at next CLI version we're using the official GraalVM gradle pluggin to build native binaries.

**Comment by drpatelh** [comment]
If this is ready to be merged then I can work off the one on `master` and update as I go through it.

**Comment by evanfloden** [comment]
Any reason to not review and edit on this branch?

**Comment by drpatelh** [comment]
> Any reason to not review and edit on this branch?

Sure, I can do that. Will create a PR relative to `readme-release-review` then.

File: README.md [hunk 6]
```diff
@@ -1,164 +1,222 @@
-# Tower CLI app
+Nextflow Tower CLI
+==================
 
-Proof of concept of Tower command line client app based on Java native compilation, Micronaut HTTP declarative client and Tower domain classes
+`tw` is Nextflow Tower on the command line. It brings Tower concepts including Pipelines, Actions and Compute Environments to the terminal.
 
-## Installation
+![tw](/tw-screenshot.png)
```
Comment (drpatelh) [inline README.md:6]: Can we create this image relative to the `rnaseq` pipeline and not `nanoseq`? Be good to be consistent in terms of which pipeline we use as our "gold standard"?

File: README.md [hunk 7]
```diff
@@ -1,164 +1,291 @@
...
+`tw` is [Tower](https://landing.tower.nf/) on the command line. It brings Tower concepts including Pipelines, Actions and Compute Environments to the terminal.
 
-## Installation
+Tower (a.k.a Nextflow Tower) is a full-stack application for the management of data pipelines and compute resources. It enables collaborative data analysis at scale, on-premises or in any cloud infrastructure.
 
-From the last Github action at artifacts section, download the binary for your OS. Unzip it, give it execution permissions with `chmod +x ./tw` and move it into a folder that it's in your path (ex: `sudo mv ./tw /usr/local/bin/tw`)
+The Tower CLI (a.k.a Nextflow Tower CLI) interacts with instances of Tower, providing an interface to launch pipelines, manage cloud resources and administer the application directly from the terminal.
 
-## Build binary development version
+![tw](assets/img/tw-screenshot.png)
```
Comment (drpatelh) [inline README.md:9]: Would be great if you could regenerate this image relative to the `rnaseq` pipeline and not `nanoseq` @evanfloden? Be good to be consistent in terms of which pipeline we use as our "gold standard"?

**Review by jordeu — APPROVED** [review]
@evanfloden for me this is good to merge. There is only one @drpatelh comment about the screenshot. But if you don't have time to update it now, tell me and I'll merge as it is now. We can leave this improvement for the next release, because we want to do a release tomorrow and it will be nice to have and updated readme. Thanks.

**Comment by drpatelh** [comment]
I have pinged Evan on Slack @jordeu ! Let's get this in!

**Comment by evanfloden** [comment]
Thanks @jordeu and @drpatelh, I'll do a final review and merge to master later today

**Pull request merged**